### DISCUSSION TITLE
Shapefile Support?

### DISCUSSION THREAD

**Discussion by gustgeo** [davenquinn/Mapboard-GIS#14] [Q&A] [answered]
Much needed application to draw maps efficiently with a pencil! Fantastic!

I can bring my rasters in as MBTiles, I can make my map, but anyway to export my work as the typical GIS shapefiles with attribute tables and such so it integrates back to my QGIS database?

**Comment by davenquinn** [comment, marked as answer]
Hi @gustgeo, thanks for the question. Mapboard GIS natively interoperates with QGIS and other GIS software. Its project file format is a Spatialite file (within a package folder).  See [the documentation](https://mapboard-gis.app/docs/projects/file-format) for more information. That means that the project can be natively opened and read (and even written to) in QGIS. It is easy to export the data to shapefiles using QGIS, `ogr2ogr` and other standard tools.

In fact, there are even ways to make the Mapboard GIS file read/write the same database as QField on your iPad. But that is not quite fully ready to go yet.
